client/admin/metric_client.go

```diff
@@ -63,7 +63,7 @@ func (c *metricClient) startMetricsRecording(
 	operation string,
 ) (metrics.MetricsHandler, time.Time) {
 	caller := headers.GetCallerInfo(ctx).CallerName
-	handler := c.metricsHandler.WithTags(metrics.OperationTag(operation), metrics.NamespaceTag(caller))
+	handler := c.metricsHandler.WithTags(metrics.OperationTag(operation), metrics.NamespaceTag(caller), metrics.ServiceRoleTag(metrics.AdminRoleTagValue))
```

Comment:
so `ServiceNameTag` and `ServiceRoleTag` are different... 😵‍💫

Reply:
Yes. I just realized we have the service role tag.

Reply:
is it possible to remove this tag? this is confusing...

Reply:
I can. But this will break existing metrics/dashboard.